js/modules/loki_message_api.js

```diff
@@ -208,19 +215,24 @@ class LokiMessageAPI {
     };
 
     const doRequest = async (nodeUrl, nodeData) => {
-      const url = `${nodeUrl}${this.messageServerPort}/retrieve`;
+      const url = `${nodeUrl}${this.messageServerPort}/v1/storage_rpc`;
+      const body = {
+        method: 'retrieve',
+        params: {
+          pubKey: ourKey,
+          lastHash: nodeData.lastHash,
```

Comment:
nodeData.lastHash can be undefined I think, does that matter?

Reply:
It would be ignored by `JSON.stringify`. And it's optional for the storage server so it's fine.

Reply:
`It would be ignored by JSON.stringify`
:+1: